docs/claws-mail.md

```diff
@@ -0,0 +1,72 @@
+# vdirsyncer with Claws Mail
+
+First of all, you must know that you can **not** edit CardDav contacts only **watch** CardDav contacts with Claws-Mail
+
+## Preparation
+First of al you need to install vdirsyncer, for that look [here](https://vdirsyncer.pimutils.org/en/stable/installation.html).
+Then we need to create some folders:
+
+    mkdir -p ~/.vdirsyncer/status
+    mkdir ~/.contacts
+
+## Configuration
+Now we create the configuration for the vdirsyncer:
```

Comment:
No is it not. But @untitaker asked me to create a HowTo for that, so that is the reason why I fully wrote it down.